go-controller/pkg/ovn/controller/services/node_tracker.go

```diff
@@ -147,28 +164,47 @@ func (nt *nodeTracker) updateNodeInfo(nodeName, switchName, routerName string, n
 	}
 
 	nt.nodes[nodeName] = ni
+	if chassisID != "" {
```

Comment:
A note here: the only change we are contemplating is that a node that did not have a chassis ID, gets one. So we are not handling whether a chassis ID was changed or removed, not even out of precaution. Just want to have a second thought about this, but if we are fine with it, all good. Perhaps a comment would be good in any case.

Reply:
This code takes care of new/updated chassisID.  We weren't cleaning up stale ones but now I fixed that.

Reply:
@dceara not sure I explained myself here. I meant that we are not clearing up old chassisID from the node tracker's `nt.nodeIPv4Template` and/or `nt.nodeIPv6Template` variables, so on every resync we could possibly be creating stale chassis variables that yes, would be cleaned up later on. But does it make sense to be constantly creating stale chassis id variables in that case? I guess we could just remove the old chassisID from the variables?

Reply:
Fixed now, by syncing all node template info to the svc controller on every relevant node change.